server/zoom/zoom.go

```diff
@@ -0,0 +1,139 @@
+package zoom
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"server/middleware"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+type MeetingOptions struct {
+	Topic       string         `json:"topic"`        //Topic
+	Type        int            `json:"type"`         // 2 for scheduled meeting
+	PreSchedule bool           `json:"pre_schedule"` //True
+	StartTime   string         `json:"start_time" `
+	Duration    int            `json:"duration"`     //120 minutes
+	Email       string         `json:"schedule_for"` //email
+	TimeZone    string         `json:"timezone"`
+	Password    string         `json:"password"` // Max 10 characters. [a-z A-Z 0-9 @ - _ *]
+	Agenda      string         `json:"agenda"`
+	Settings    MeetingSetting `json:"settings"`
+}
+type MeetingSetting struct {
+	HostVideo        bool   `json:"host_video"`
+	ParticipantVideo bool   `json:"participant_video"`
+	ChinaMeeting     bool   `json:"cn_meeting"`
+	IndiaMeeting     bool   `json:"in_meeting"`
+	JoinBeforeHost   bool   `json:"join_before_host"`
+	JBHTime          int    `json:"jbh_time"`
+	MuteUponEntry    bool   `json:"mute_upon_entry"`
+	Watermark        bool   `json:"watermark"`
+	UsePMI           bool   `json:"use_pmi"`        //false
+	ApprovalType     int    `json:"approval_type"`  // 2
+	Audio            string `json:"audio"`          //both
+	AutoRecording    string `json:"auto_recording"` //local
+	//EnforceLogin                 bool   `json:"enforce_login"`  //true
+	WaitingRoom bool `json:"waiting_room"`
+	//RegistrantsConfirmationEmail bool   `json:"registrants_confirmation_email"`
+}
+
+type ZoomLink struct {
+	JoinLink  string `json:"join_url"`
+	StartLink string `json:"start_url"`
+}
+
+func CreateZoomLink(appointment middleware.Appointment) (string, string, error) {
+	jwtToken, _ := jwtToken(os.Getenv("API_KEY"), os.Getenv("API_SECRET"))
```

Comment:
These things should be constants like the google key in handlers.go. Just fyi, if you create environment variables though, add them to run.sh and update run.sh